src/git-pull-request

```diff
@@ -2,25 +2,15 @@
 source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helpers/helpers.sh"
 
 
-# Returns the source for a new Bitbucket pull request
-function bitbucket_source {
-  sha=$(git log --format="%H" -1 | cut -c-12)
-  echo "$repository:$sha:$branch" | sed 's/\//%2F/g' | sed 's/\:/%3A/g'
+function preconditions {
+  activate_driver_family 'code_hosting' \
+                         'Pull requests can only be created on GitHub or Bitbucket.'
```

Comment:
I think this error belongs in `src/drivers/code_hosting/activate.sh`
This code shouldn't have to know what drivers are supported